1Homework18.02.22/SquareSorting/SquareSorting/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace SquareSorting;
+internal class Program
```

Comment:
```suggestion
namespace SquareSorting;

internal class Program
```
А ещё самим классам тоже нужны комментарии